build.ps1

```diff
@@ -82,13 +94,16 @@ function Get-ProjectBuildDir($Arch, $ID)
   return "$BinaryCache\" + ($Arch.BuildID + $ID)
 }
 
+# We need to use different trees for the platform directory because the path to Swift
+# modules .lib files would clash across architectures.
+function Get-ArchSpecificPlatformDir($Arch, $SubDir = "")
+{
+  return [IO.Path]::Combine("$BinaryCache\Windows.platform\$($Arch.ShortName)", $SubDir)
```

Comment:
I'd rather have the architecture be hoisted above the platform if you want to do this.

Reply:
Done